style.css

```diff
@@ -315,7 +326,7 @@ a {
   margin-left: 150px;
 }
 
-.dropBtn2 {
+#dropBtn2 {
```

Comment:
- [ ] [OPTIONAL] I noticed this part is not being displayed properly in mobile view (contact us section), It would be great if we can fix that I just wanted to let you be aware of the problem 

![image](https://user-images.githubusercontent.com/87689547/157380658-4865475e-2a83-4144-a150-a77806b0104e.png)

Reply:
Thank you, I did that